src/ru/2018-01/08/03.md

```diff
@@ -1,7 +1,22 @@
 ---
-title:  Урок
+title:  Божьи благословения
 date:   19/02/2018
 ---
 
-### <center>Мы подготавливаем данный урок</center>
-<center>Попробуйте позже</center>
\ No newline at end of file
+В `Мал. 3:10` Бог обещает великие благословения тем, кто будет верен в принесении десятин. Однако Божье благословение имеет не одно, а множество измерений. Если мы сводим все благословения к накоплению материальных средств, то не понимаем характера Божьих благословений.
```

Comment:
Please remove all ``` ` ``` around Bible verses, because it will create comment box for the answers